### DISCUSSION TITLE
Is there a public Roadmap / Release schedule of Talos?

### DISCUSSION THREAD

**Discussion by mstaal** [siderolabs/talos#8951] [General]
I have been looking for some kind of roadmap / release schedule of Talos. Does such a thing exist, or could it be requested? I am looking forward to v1.8.0 which should (according to something I read on Reddit) include better utilisation of volumes, but I think in general it would be nice to more easily read into what's going on, and when to expect new releases. Do you have release notes as well?

**Comment by steverfrancis** [comment]
It doesn't currently exist, but should, and we are exploring how to make it so.
(Probably to start with by making our internal planning project board public.)

We do have release notes - they are in every release of Talos on Github.
e.g. https://github.com/siderolabs/talos/releases
You can subscribe for notifications on that release page.

**Comment by smira** [comment]
Every Talos future release has some very high-level roadmap: https://github.com/siderolabs/talos/issues/8484
